### ISSUE TITLE
SVG preview only works without "@" syntax

### ISSUE THREAD

**Issue by MinnieTheMoocher** [kisstkondoros/gutter-preview#36]
SVGs currently only can properly be previewed when referenced
via "../../"... pathes, but not when using alias pathes from tsconfig.json.

Example:

this preview works:
```
&-start {
            background-image: url("../../../assets/buttons/Destination.svg");
            background-size: contain;
            background-position: center;
            @include size(32px, 32px);
         }
```

this preview does not work:
```
 &-dest {
            background-image: url("@assets/buttons/Destination.svg");
            background-size: contain;
            background-position: center;
            @include size(32px, 32px);
         }

```

Expected: also the "@" syntax variant works properly

See attached screenshot:
- green = good case
- pink = broken case

![screenshot](https://user-images.githubusercontent.com/3216359/52426822-01660100-2aff-11e9-9253-ed4f3d1f4599.jpg)

**Comment by kisstkondoros** [comment]
Thanks for reaching out!
In #21 this already came up, I will check it again.

**Comment by kisstkondoros** [comment]
The feature is available in version 0.17.1 :fireworks:

**Comment by canvascat** [comment]
but not work with `@` in v0.30.0

![image](https://user-images.githubusercontent.com/31235016/236596152-7e6ca618-e391-44d1-88a0-467a9e490b02.png)
